.gitattributes

```diff
@@ -1,2 +1,3 @@
-spec/fixtures/vcr_cassettes -diff
+*.rb diff=ruby
+spec/fixtures/vcr_cassettes -diff -merge
```

Comment:
Yea, I'm not sure it's going to achieve the goal of removing them from github diffs.  Wondering whether or not to remove the file....